src/Find.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Find (findMatches, recipeToString) where
+
+import Types
+import RichText (toStr)
+import Text.Regex.TDFA ((=~))
+
+findMatches :: String -> [Recipe] -> [String]
+findMatches rgx = concatMap zipName 
+    where zipName r = 
+            let name = recipeName r 
+             in zipWith (\a b -> a ++ ": " ++ b) (repeat name) . findInRecipe rgx $ r
```

Comment:
I don't normally see the `in` of a `let` expression indented one more space - Could you re-indent it at the same level as the `let`?

Reply:
Sure thing, I usually indent `in` one space so that the following statement lines up vertically with the `let` blocks. Fixed.